src/Spryker/Client/ProductCategoryStorage/ProductCategoryStorageClientInterface.php

```diff
@@ -13,6 +13,7 @@ interface ProductCategoryStorageClientInterface
      * Specification:
      * - Returns Product Abstract Category by id.
      * - Forward compatibility (from next major): only product abstract categories assigned with passed $storeName will be returned.
+     * - Expands Product Abstract Category by {@link \Spryker\Client\ProductCategoryStorageExtension\Dependency\Plugin\ProductAbstractCategoryStorageCollectionExpanderPluginInterface} plugin stack execution.
```

Comment:
core PR annotation differs from this one

Reply:
it's probably some GitHub issue since this was already deleted.